lib/mongo/client_encryption.rb

```diff
@@ -53,6 +53,8 @@ def initialize(key_vault_client, options = {})
     #     by a colon (e.g. "kms.us-east-1.amazonaws.com" or
     #     "kms.us-east-1.amazonaws.com:443"). An endpoint in any other format
     #     will not be properly parsed.
+    # @option [ Array ] :key_alt_names An optional array of strings specifying
```

Comment:
` @option options [ Array<String> ] `

Reply:
There's also the "options" between @option and the type.

Reply:
I've been declaring these incorrectly 😆Just fixed all of them.

Reply:
:+1: